test/integration/__snapshots__/integration.test.ts.snap

```diff
@@ -429,3 +429,44 @@ export const tables = {
 }
 "
 `;
+
+exports[`Integration tests Schema generation prefixWithSchemaNames generation 1`] = `
+"/* tslint:disable */
+/* eslint-disable */
+
+
+export type Json = unknown;
+export type CommentStatusEnum = 'archived' | 'complete' | 'deleted' | 'pending';
+
+// Table public_Users
+export interface PublicUsers {
+  id: string;
+  name: string;
+  pronoun: string | null;
+}
+export interface PublicUsersInput {
+  id?: string;
+  name: string;
+  pronoun?: string | null;
+}
+const public_Users = {
```

Comment:
This variable name looks a bit strange. Can we make it either `public_users` or `publicUsers`? The former feels more consistent with the existing un-prefixed names.